cardano-testnet/src/Cardano/Testnet.hs

```diff
@@ -13,6 +13,8 @@ module Cardano.Testnet (
   TestnetNodeOptions(..),
   cardanoDefaultTestnetOptions,
   cardanoDefaultTestnetNodeOptions,
+  getDefaultAlonzoGenesis,
```

Comment:
Why do we need those exports?

Reply:
Because this file is used as an API from tests files. Tests files don't import `Testnet.Start.Cardano` directly, they import this file. E.g. compare:

```shell
→ git grep 'import.*Cardano.Testnet' | wc -l
44
```

and

```shell
→ git grep 'import.*Testnet.Start.Cardano' | wc -l
3
```

Reply:
But how do you use genesis functions in tests?

Reply:
In the same manner as calling `cardanoTestnetDefault`, see e.g. [a caller](https://github.com/IntersectMBO/cardano-node/blob/master/cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/SanityCheck.hs#L54), and the [corresponding import](https://github.com/IntersectMBO/cardano-node/blob/master/cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/SanityCheck.hs#L15).